server/tests/common/backend.py

```diff
@@ -100,14 +100,17 @@ def app(backend: Backend, monkeypatch: pytest.MonkeyPatch) -> Iterator[AsgiApp]:
     yield asgi_app
 
 
+type LoadedTemplate = dict[str, Template]
```

Comment:
It seems to me that there is nothing special about "loaded". I'd just add `Dict` suffix (if `Template` is renamed in `TestbedTemplate`):
```suggestion
type TestbedTemplateDict = dict[str, TestbedTemplate]
```

Reply:
done